### PR TITLE
[PM-15112] Fix missing i18n and block ssh key creation in web

### PR DESCRIPTION
**Pull request by quexten**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/PM-15112

## 📔 Objective

Since web is still missing capabilities for generating keys, we detect if the add-edit component gets opened with the ssh-key cipher-type selected and force switch to login, so that we do not accidentally allow the user to create broken ssh key ciphers.

This wil be removed as soon as we can generate ssh keys via the sdk in web.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: apps/web/src/app/vault/individual-vault/add-edit.component.ts [hunk 1]
```diff
@@ -100,6 +100,14 @@ export class AddEditComponent extends BaseAddEditComponent implements OnInit, On
   async ngOnInit() {
     await super.ngOnInit();
     await this.load();
+
+    // https://bitwarden.atlassian.net/browse/PM-10413
+    // cannot generate ssh keys so block creation
+    if (this.type == CipherType.SshKey && this.cipherId == null) {
```
Comment (Jingo88) [inline apps/web/src/app/vault/individual-vault/add-edit.component.ts:106]: Can we make this `===`

**Review by Jingo88 — CHANGES_REQUESTED** [review]

File: apps/web/src/app/vault/individual-vault/add-edit.component.ts (hunk 1 above)
Comment (quexten) [reply apps/web/src/app/vault/individual-vault/add-edit.component.ts:106]: Ok, but this needs to be `=== undefined` then.

Comment (Jingo88) [reply apps/web/src/app/vault/individual-vault/add-edit.component.ts]: oh apologies. I meant `===` for the `this.type` haha. The `this.cipherId == null` is fine.

Comment (quexten) [reply apps/web/src/app/vault/individual-vault/add-edit.component.ts]: Ohh, whoops. Fixed now.

**Pull request merged**